docs/api/ReactWrapper/contains.md

```diff
@@ -6,7 +6,7 @@ It will determine if an element in the wrapper matches the expected element by c
 
 #### Arguments
 
-1. `nodeOrNodes` (`ReactElement|Array<ReactElement>`): The node or array of nodes whose presence you are detecting in the current instance's
```

Comment:
Even though it's a list of 1, it's still an ordered list. This is consistent with the rest of the docs as well.